### PR TITLE
knollfear/bcda-894

### PR DESCRIPTION
**Pull request by knollfear**
### Fixes [BCDA-894](https://jira.cms.gov/browse/BCDA-894)

Elissa provided me with the shared token for open sandbox to add to the user guide.
I also added a copy to clipboard button to the token and CURL snippet to ease the use of these features.  
Noticed that on a small screen the CTA buttons touched so I added some margin to make it look better.
### Security Implications

Added the public token for Open Sandbox.  This is a known thing and not a security threat.

### Acceptance Validation

copy to clipboard works
Token workson open sandbox
cta buttons don't touch on small screens

### CODE REVIEW

File: bcda-site-static/_includes/copy_snippet.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% assign code = include.code %}
+
+<div class="highlight"><pre class="highlight"><code>{{code}}
+</code></pre></div>
+
+{% assign nanosecond = "now" | date: "%N" %}
+<input type="text" id="code{{ nanosecond }}" style="position: absolute;left: -1000px;" value="{{ code }}"/>
+<div class="button" on>
+    <p>
+        <a href="javascript:void()" onclick="copyText{{ nanosecond }}()" id="copybutton{{ nanosecond }}">
```
Comment (tbellj) [inline bcda-site-static/_includes/copy_snippet.md]: just looked it up, is this supposed to be void(0)?

File: bcda-site-static/user_guide.md [hunk 2]
```diff
@@ -62,9 +69,12 @@ Metadata about the Beneficiary Claims Data API is available as a FHIR [Capabilit
 `GET /api/v1/metadata`
 
 ###### cURL command
-```sh
+{%- capture code -%}
 curl https://sandbox.bcda.cms.gov/api/v1/metadata
-```
+{%- endcapture -%}
```
Comment (tbellj) [inline bcda-site-static/user_guide.md]: can't remember what these tags do..

Comment (knollfear) [reply bcda-site-static/user_guide.md]: captures the text inside as the variable "code"

File: bcda-site-static/_includes/copy_snippet.md (hunk 1 above)
Comment (knollfear) [reply bcda-site-static/_includes/copy_snippet.md]: you are right.  I fixed it

**Pull request merged**